package.json

```diff
@@ -43,6 +45,8 @@
   "dependencies": {
     "@carbon/react": "^1.33.1",
     "@hookform/resolvers": "^3.3.0",
+    "@playwright/test": "^1.47.2",
```

Comment:
Use the latest version of Playwright, checkout release notes at https://playwright.dev/docs/release-notes

Reply:
Thanks for the suggestion, @kdaud! I've updated Playwright to the latest version and reviewed the release notes. All looks good so far.